test/lit.cfg

```diff
@@ -1439,24 +1439,28 @@ config.substitutions.append(('%module-target-future', target_future))
 # Add 'target-sdk-name' as the name for platform-specific directories
 config.substitutions.append(('%target-sdk-name', config.target_sdk_name))
 
+path_separator = "/"
+if platform.system() == 'Windows':
+   path_separator = "\"
+
 # Add 'stdlib_dir' as the path to the stdlib resource directory
 stdlib_dir = config.swift_lib_dir + "/swift/"
-if platform.system() == 'Linux':
-    stdlib_dir += config.target_sdk_name + "/" + run_cpu
+if platform.system() == 'Linux' || platform.system() == 'Windows':
+    stdlib_dir += config.target_sdk_name + path_separator + run_cpu
```

Comment:
Using `os.path.join` is better IMO, though you could just keep this as is I _think_.

Reply:
A lot better, thanks for the pointer.